src/Nethermind/Nethermind.Core/Crypto/Keccak.cs

```diff
@@ -33,6 +33,12 @@ public unsafe struct ValueKeccak
         /// </returns>
         public static readonly ValueKeccak OfAnEmptyString = InternalCompute(new byte[] { });
         
+        public static readonly ValueKeccak EmptyTreeHash = InternalCompute(new byte[] {128});
+
+        public ValueKeccak(Span<byte> bytes)
+        {
+            bytes.CopyTo(BytesAsSpan);
```

Comment:
Allocations I managed to avoid! This is I didn't use ValueKeccak. I don't want to create new independent entities. I just wanted to create a strongly typed view over big byte array.

Reply:
where do you have allocations here?